tests/e2e/performance/test_pvc_bulk_clone_performance.py

```diff
@@ -0,0 +1,196 @@
+            pvc_dict_list, clone_yaml, sc_name
+        )
+
+        logging.info("Created clone dict list")
+
+        job_clone_file = ObjectConfFile(
+            name="job_profile_clone",
+            obj_dict_list=clone_dict_list,
+            project=self.namespace,
+            tmp_path=tmp_path,
+        )
+
+        # Create kube_job that creates clones
+        job_clone_file.create(namespace=self.namespace)
+
+        logging.info("Going to check bound status for clones")
+        # Check all the clones reached Bound state
+        clone_bound_list = scale_lib.check_all_pvc_reached_bound_state_in_kube_job(
+            kube_job_obj=job_clone_file,
+            namespace=self.namespace,
+            no_of_pvc=pvc_count,
+            timeout=600,
+        )
+
+        logging.info(f"Number of clones in Bound state {len(clone_bound_list)}")
+
+        clone_objs = []
+        all_pvc_objs = pvc.get_all_pvc_objs(namespace=self.namespace)
+        for clone_yaml in clone_dict_list:
+            name = clone_yaml["metadata"]["name"]
+            size = clone_yaml["spec"]["resources"]["requests"]["storage"]
+            logging.info(f"Clone {name} of size {size} created")
+            for pvc_obj in all_pvc_objs:
+                if pvc_obj.name == name:
+                    clone_objs.append(pvc_obj)
+
+        assert len(clone_bound_list) == len(
+            clone_objs
+        ), f"There are object clones not found, cannot measure time"
+        start_time = helpers.get_provision_time(
+            self.interface, clone_objs, status="start"
+        )
+        end_time = helpers.get_provision_time(self.interface, clone_objs, status="end")
+        total_time = (end_time - start_time).total_seconds()
+        speed = round(total_files_size / total_time, 2)
+        logging.info(
+            f"Total creation time = {total_time} secs, data size = {total_files_size} MB, speed = {speed} MB/sec"
+        )
+
+    def run_fio_on_pvcs(self, pvc_dict_list, pod_factory):
+        total_files_size = 0
+        all_pvc_objs = pvc.get_all_pvc_objs(namespace=self.namespace)
+        logging.info(f"Found {len(all_pvc_objs)} PVCs")
+        for pvc_yaml in pvc_dict_list:
+            pvc_name = pvc_yaml["metadata"]["name"]
+            pvc_size = pvc_yaml["spec"]["resources"]["requests"]["storage"]
+            logging.info(f"Size of pvc {pvc_name} is {pvc_size}")
+            pvc_size_int = int(pvc_size[: len(pvc_size) - 2])  # without "Gi"
+            file_size_mb = int(pvc_size_int * 0.6) * constants.GB2MB
+            total_files_size += file_size_mb
+            file_size_mb_str = str(file_size_mb) + "M"
+            logging.info(f"Writing file of size {file_size_mb_str}")
+
+            # now find pvc_obj by name and create pod_obj to write to
+            pvc_obj = None
+            for obj in all_pvc_objs:
+                if obj.name == pvc_name:
+                    pvc_obj = obj
```

Comment:
it is better to remove the object that found from the total list, so it will reduce iterations for next time.

Reply:
@Avilir you are right, fixed the test, thank you!